src/js/tech/ChromecastTech.js

```diff
@@ -143,6 +157,54 @@ ChromecastTech = {
       this._playSource(source, 0);
    },
 
+   /**
+    * transform trackdata in cast Track
```

Comment:
Please write as a sentence (capitalize first letter, add period at end) and fix capitalization of words in the sentence.

Reply:
https://github.com/silvermine/videojs-chromecast/pull/112/commits/6c92f8c8305f06c5c07392c7cb7101fe178dfc21